extensions/amp-consent/0.1/consent-policy-manager.js

```diff
@@ -171,22 +171,36 @@ export class ConsentPolicyInstance {
       return;
     }
 
-    if (state != CONSENT_ITEM_STATE.DISMISSED) {
-      this.itemToConsentState_[consentId] = state;
-    } else {
+
+    if (state == CONSENT_ITEM_STATE.IGNORED) {
+      // Ignore the consent item state
+      if (this.itemToConsentState_[consentId] != CONSENT_ITEM_STATE.GRANTED &&
```

Comment:
It can be hard to figure out what state we are in, maybe descriptive variables like `const shouldOverwrite === this.itemConsentState ... && ... ` would help, or maybe more comments

Reply:
or maybe helper functions like you mentioned

Reply:
added a `shouldOverwrite` const